website/sidebars.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
                 'guides/request-storage',
                 'guides/result-storage',
                 'guides/configuration',
+                'guides/http-crawler-guide',
```

Comment:
this should go to a less prominent place, i'd put it after the got scraping guide maybe